pkg/cri/resource-manager/policy/builtin/topology-aware/pools.go

```diff
@@ -391,7 +391,10 @@ func (p *policy) allocatePool(container cache.Container, poolHint string) (Grant
 			request, supply.DumpAllocatable(), err)
 	}
 
-	log.Debug("allocated req '%s' to memory node '%s' (memset %s,%s)", container.GetCacheID(), grant.GetMemoryNode().Name(), grant.GetMemoryNode().GetMemset(memoryDRAM), grant.GetMemoryNode().GetMemset(memoryPMEM))
+	log.Debug("allocated req '%s' to memory node '%s' (memset %s,%s)",
```

Comment:
Should we be prepared for HBM, too(?)

Reply:
Yes, since the code does have preliminary support for HBM, maybe we could dump that here, too...

Reply:
Added.